sympy/solvers/solveset.py

```diff
@@ -871,3 +872,295 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def linear_eq_to_matrix(equations, *symbols):
+    r"""
+    Converts a given System of Equations into Matrix form.
+    Here `equations` must be a linear system of equations in
+    `symbols`. The order of symbols in input `symbols` will
+    determine the order of coefficients in the returned
+    Matrix.
+
+    The Matrix form corresponds to the augmented matrix form.
+    For example:
+
+    4.x + 2.y + 3.z  = 1
```

Comment:
> For docstrings I try to promote non-latex heavy equations so that they are readable from the terminal. I don't think SymPy has consensus on this but I find this much more preferable.

Just for the record:
I see your point, and I tend to avoid LaTeX myself because I'm having 
trouble reading and writing it in source form.
However, I think the ASCII-compatible formulae can be trivially 
translated to LaTeX by placing them between $$, and people will learn to 
"see through" these just as the "see through" backticks that indicate 
Python objects, I think that's more than outweighed if we can 
consistently typeset formulae. Obviously, YMMV :-)